README.md

```diff
@@ -1,243 +1,64 @@
-# Cap'n Proto bindings for Node.js
+# `node-capnp`
```

Comment:
Moving the readmes around like this is likely to cause much faff when attempting to merge changes from upstream, hence why our own readme was previously added  as a separate file. By all means we can add a pointer to our readme to the upstream readme, but I don't think we should move them around like this.

Reply:
It took several months for me to realise that `README-FOURIER.md` existed at all, which is why I leaned hard towards maximum visibility here - I'd expect our readme to be read much more often than it's edited.

That being said, it looks like Git's rename detection has already failed us here, so editing a link into the original readme does sound like a pretty fair compromise.